PdfScribeInstallCustomAction/PdfScribeInstaller.cs

```diff
@@ -38,10 +38,10 @@ public class PdfScribeInstaller
         private readonly String logEventSourceNameDefault = "PdfScribeCore";
 
         const string ENVIRONMENT_64 = "Windows x64";
-        const string PRINTERNAME = "PDF Scribe";
-        const string DRIVERNAME = "PDF Scribe Virtual Printer";
-        const string HARDWAREID = "PDFScribe_Driver0101";
-        const string PORTMONITOR = "PDFSCRIBE";
+        const string PRINTERNAME = "Aivika Printer";
+        const string DRIVERNAME = "Aivika Virtual Printer";
+        const string HARDWAREID = "Aivika_Driver0101";
+        const string PORTMONITOR = "Aivika";
         const string MONITORDLL = "redmon64pdfscribe.dll";
         const string PORTNAME = "PSCRIBE:";
```

Comment:
Would renaming this break anything?

Reply:
I not sure about it, I never try to change that before. 
It is just naming, I think should not be a problem.